tests/queries/0_stateless/02718_insert_meet_hardware_error.sql

```diff
@@ -0,0 +1,20 @@
+-- Tags: zookeeper
```

Comment:
`system enable failpoint` isn't restricted to schema-scope, right? in that case better disable parallel execution of the test via `no-parallel` tag.